pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -610,14 +612,23 @@ func validateValidatingAdmissionPolicySpec(spec *admissionregistration.Validatin
 			allErrors = append(allErrors, field.Required(fldPath.Child("matchConstraints", "resourceRules"), ""))
 		}
 	}
-	if len(spec.Validations) == 0 {
-		allErrors = append(allErrors, field.Required(fldPath.Child("validations"), ""))
+	if len(spec.Validations) == 0 && len(spec.AuditAnnotations) == 0 {
+		allErrors = append(allErrors, field.Required(fldPath, "one of validations and auditAnnotations is required"))
```

Comment:
one of:
* emit for validations field
* emit for both validations and auditAnnotations (maybe this is best?)
* emit invalid field error on the parent